src/System.Security.Cryptography.Xml/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -1,2 +1,9 @@
 # Type must be public in implementation for serialization to work but we don't want to expose it publicly in the contract as it isn't public on .NET Framework
 TypesMustExist : Type 'System.Security.Cryptography.Xml.CryptoSignedXmlRecursionException' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'System.Security.Cryptography.Xml.XmlDsigRSASHA256Url' does not exist in the reference but it does exist in the implementation.
```

Comment:
@weshaggard Thoughts / feelings on API that was added in net462 for a netstandard20 contract?  I know that some stuff got grandfathered in by being part of netstandard16.

Plus, it's const values... so even if someone does run it against net461 I don't think there'd be any exceptions from it.

Reply:
Assuming this contract only exists inbox on .NET Framework and not anywhere else I think we are OK to expose these in the netstandard ref given they already exist in net462. 

As for the them only being const values while I agree with you that they might be allowable given the const value is hard-coded by the compiler I would still avoid including them if they weren't already on the inbox library.

Reply:
With that said I'm not sure I understand why these need to be in this baseline file as it looks like they were added to the netstandard implementation configuration as well as the contract so there shouldn't be an error.

Reply:
@weshaggard The simplest answer is that I may have misunderstood what should go into ref and what should not :-)

Reply:
@filipnavara can you try undo'ing your change to this baseline file and seeing if your build passes?

Reply:
Yep, in a minute.

Reply:
Seems to work.